src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/TestRunner.cs

```diff
@@ -311,6 +311,18 @@ internal static void TestToString(FormulaValue result, StringBuilder sb)
 
                 sb.Append('}');
             }
+            else if (result is TimeValue tv)
+            {
+                sb.Append(tv.Value.ToString());
+            }
+            else if (result is DateValue d)
+            {
+                sb.Append(d.Value.ToString(@"MM/dd/yyyy hh:mm:ss tt", System.Globalization.CultureInfo.InvariantCulture));
```

Comment:
[nit] can we use 24h time for the tests (format specifier `HH`)? That's more "global" than using am/pm.

Reply:
The build system formats it in AM/PM, so it was done this way to pass the unit tests.

Reply:
IIRC this is used to compare with the expected value defined in the .txt test files, so if we update the value there (e.g., mathfuncs.txt, line 149), it should work.

Reply:
That's ok, we can look at fixing it later...